Zone5/Transport/RequestEndpoint.swift

```diff
@@ -1,7 +1,8 @@
 import Foundation
 
-/// Protocol defining the requirements for endpoints accepted when generating requests for the API.
-public protocol RequestEndpoint {
+/// Protocol defining internal endpoints accepted when generating requests for the API.
+/// These endpoints contain relative uris and need to be wrapped by a RequestEndpoint to add server baseURL
+public protocol InternalRequestEndpoint {
```

Comment:
Should this be private?

Reply:
If i make that internal or private I get "Public protocol cannot refine an internal protocol" for the RequestEndpoint (which extends it.

The ExternalView funcs all take RequestEndpoint so you can't incorrectly pass an InternalRequestEndpoint.